nemo_text_processing/inverse_text_normalization/zh/taggers/word.py

```diff
@@ -26,5 +26,5 @@ class WordFst(GraphFst):
 
     def __init__(self):
         super().__init__(name="word", kind="classify")
-        word = pynutil.insert('name: "') + pynini.closure(NEMO_NOT_SPACE, 1) + pynutil.insert('"')
```

Comment:
remind me on this syntax, is the closure(...,1) equivalent to 0 or 1 or match exactly one?